splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -292,6 +292,49 @@ public void createSystemViews(TransactionController tc) throws StandardException
         SpliceLogUtils.info(LOG, "Views in SYSVW created!");
     }
 
+    public void createTableColumnViewInSysIBM(TransactionController tc) throws StandardException {
+        tc.elevate("dictionary");
+
+        /**
+         * handle syscolumns in sysibm
+         */
+        // check the existence of syscolumns view in sysibm
+        TableDescriptor td = getTableDescriptor("SYSCOLUMNS", sysIBMSchemaDesc, tc);
+
+        // drop it if it exists
+        if (td != null) {
+            ViewDescriptor vd = getViewDescriptor(td);
+
+            // drop the view deifnition
+            dropAllColumnDescriptors(td.getUUID(), tc);
+            dropViewDescriptor(vd, tc);
+            dropTableDescriptor(td, sysIBMSchemaDesc, tc);
+        }
```

Comment:
Why sysibm.syscolumns and sysibm.systables can already exist?

Reply:
I'm expecting that the views may need to be revised based on users feedback, so this function will be called to update the definition. Also for force upgrade scenarios, the views may also exist.